application/src/main/java/org/opentripplanner/standalone/configure/ConstructApplicationFactory.java

```diff
@@ -90,6 +91,7 @@ public interface ConstructApplicationFactory {
   VehicleParkingRepository vehicleParkingRepository();
   VehicleParkingService vehicleParkingService();
   TimetableSnapshotManager timetableSnapshotManager();
+  TripPatternCache tripPatternCache();
```

Comment:
We can leave this for know, but I think the RT-updaters should probably have their own dagger context. The reson why I have not put some of the updators into Dagger is that we need to analyze what we should do. I think the construct-application context get to big if everything should live there.